tools/extract_cmake_functions.py

```diff
@@ -75,10 +118,9 @@
                 for match in CMAKE_FUNCTION_RE.finditer(text):
                     name = match.group(4)
                     signature = match.group(1).strip()
-                    description = match.group(2)
-                    description = description.replace('#', '').strip()
                     if signature.startswith(name):
-                        all_functions[name] = (signature, description, group)
+                        description, brief, params = process_commands(match.group(2).replace('#', ''), name, group, signature)
+                        all_functions[name] = (group, signature, brief, description, params)
```

Comment:
Probably makes sense to use a dictionary here rather than a tuple, which would simplify the file-writing loop.